src/Service.Host/config-example.env

```diff
@@ -6,11 +6,11 @@ RABBIT_SERVER=localhost
 RABBIT_PORT=5672
 RABBIT_USERNAME=guest
 RABBIT_PASSWORD=guest
-APP_ROOT=http://localhost:61798
+APP_ROOT=https://localhost:61798
 AUTHORITY_URI=https://www.linn.co.uk/auth/
-PROXY_ROOT=http://app.linn.co.uk
+PROXY_ROOT=https://app.linn.co.uk
 SMTP_HOSTNAME=smarthost
 SHIPFILES_FROM_ADDRESS=someone@something.co.uk
 SHIPFILE_TEMPLATE_PATH=./views/ShipfilePdfTemplate.html
-HTML_TO_PDF_API_CONVERSION_ENDPOINT=http://localhost:5000/convert/html
+HTML_TO_PDF_API_CONVERSION_ENDPOINT=https://localhost:5000/convert/html
```

Comment:
for when you update the real configs in S3
For some reason the sys healtcheck for the pdf service (on http and https) is bad gatewaying, but the real one works fine on https, e.g.
https://app.linn.co.uk/pdf-service/ping
so this shouldn't break anything

Reply:
i've already switched stores over in real life. Now if anyone links out to an non https friendly solution they'll have to go and fix it first!